Model/IdealPaymentMethod.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use Wirecard\PaymentSdk\Config\Config;
+use Wirecard\PaymentSdk\Config\PaymentMethodConfig;
+use Wirecard\PaymentSdk\Entity\IdealBic;
+use Wirecard\PaymentSdk\Transaction\IdealTransaction;
+use Wirecard\PaymentSdk\Transaction\Transaction;
+
+use Wirecard\Oxid\Core\Helper;
+
+use OxidEsales\Eshop\Core\Registry;
+
+/**
+ * Payment method implementation for iDEAL
+ *
+ * @since 1.2.0
+ */
+class IdealPaymentMethod extends SepaCreditTransferPaymentMethod
+{
+    /**
+     * @inheritdoc
+     *
+     * @since 1.2.0
+     */
+    protected static $_sName = "ideal";
+
+    /**
+     * @inheritdoc
+     *
+     * @var bool
+     *
+     * @since 1.2.0
+     */
+    protected static $_bMerchantOnly = false;
```

Comment:
Since most payment methods are not merchant only we might want to set the default to `false`.

Reply:
we do, but this method inherits from sepa ct and there it is set to true

Reply:
Nevermind, I was under the impression that Sepa CT was an abstract class.